core/src/main/java/com/yahoo/oak/common/intbuffer/OakIntBufferComparator.java

```diff
@@ -39,13 +39,15 @@ public int compareKeyAndSerializedKey(ByteBuffer key, OakScopedReadBuffer serial
         return compare(key, 0, size, unsafeKey.getByteBuffer(), unsafeKey.getOffset(), size);
     }
 
-    public static int compare(ByteBuffer buff1, int pos1, int size1, ByteBuffer buff2, int pos2, int size2) {
```

Comment:
separate PR (?) - adding final in multiple locations

Reply:
ok. will add a separate PR for the formatting/simple changes.